tests/alarms/models/test_holds_alarms.py

```diff
@@ -0,0 +1,195 @@
+            "hold_info_test_redshift_db", "2023-06-01"
+        )
+        mock_modified_query.assert_called_once_with("hold_info_test_redshift_db")
+        mock_null_query.assert_called_once_with(
+            "hold_info_test_redshift_db", "2023-06-01"
+        )
+        test_instance.redshift_client.execute_query.assert_has_calls(
+            [
+                mocker.call("count query"),
+                mocker.call("count query"),
+                mocker.call("deleted query"),
+                mocker.call("modified query"),
+                mocker.call("null query"),
+            ]
+        )
+        test_instance.redshift_client.close_connection.assert_called_once()
+
+    def test_run_checks_no_holds_records(self, test_instance, mocker, caplog):
+        mocker.patch("alarms.models.holds_alarms.build_redshift_holds_deleted_query")
+        mocker.patch("alarms.models.holds_alarms.build_redshift_holds_modified_query")
+        mocker.patch("alarms.models.holds_alarms.build_redshift_holds_null_query")
+        mocker.patch("alarms.models.holds_alarms.build_redshift_holds_query")
+        test_instance.redshift_client.execute_query.side_effect = [
+            ([0],),
+            ([10],),
+            (),
+            (),
+            (),
+        ]
+
+        with caplog.at_level(logging.ERROR):
+            test_instance.run_checks()
+        assert (
+            '"hold_info_test_redshift_db" table not updated for all of '
+            "2023-05-31 (ET)"
+        ) in caplog.text
+
+    def test_run_checks_no_holds_queue_records_alarm(
+        self, test_instance, mocker, caplog
+    ):
+        mocker.patch("alarms.models.holds_alarms.build_redshift_holds_deleted_query")
+        mocker.patch("alarms.models.holds_alarms.build_redshift_holds_modified_query")
+        mocker.patch("alarms.models.holds_alarms.build_redshift_holds_null_query")
+        mocker.patch("alarms.models.holds_alarms.build_redshift_holds_query")
+        test_instance.redshift_client.execute_query.side_effect = [
+            ([10],),
+            ([0],),
+            (),
+            (),
+            (),
+        ]
+
+        with caplog.at_level(logging.ERROR):
+            test_instance.run_checks()
+        assert (
+            '"queued_holds_test_redshift_db" table not updated for all of '
+            "2023-05-31 (ET)"
+        ) in caplog.text
+
+    def test_run_checks_deleted_holds_alarm(self, test_instance, mocker, caplog):
+        mocker.patch("alarms.models.holds_alarms.build_redshift_holds_deleted_query")
+        mocker.patch("alarms.models.holds_alarms.build_redshift_holds_modified_query")
+        mocker.patch("alarms.models.holds_alarms.build_redshift_holds_null_query")
+        mocker.patch("alarms.models.holds_alarms.build_redshift_holds_query")
+        test_instance.redshift_client.execute_query.side_effect = [
+            ([10],),
+            ([10],),
+            (
+                [
+                    1,
```

Comment:
This formatting seems extreme haha... I leave the linter off for test files because it often results in stuff like this. Mostly everything looks fine but do you mind cleaning up egregious line spacing like this?